portfolio/src/main/webapp/script.js

```diff
@@ -63,14 +63,23 @@ function previousExperience() {
   selectedExperience--;
 }
 
+var number = 0;
```

Comment:
Try to be more specific when naming your variables. It isn't clear what `number` is referring to here. You could rename it to something like `numComments` or even `numCommentsDisplayed` so it's clear what the value means.